portal/src/main/webapp/js/lib/oql/oql_basic.pegjs

```diff
@@ -80,15 +80,15 @@ ComparisonOp
 	/ "<" { return "<"; }
 
 Mutation
-	= "MISSENSE"i { return {"type":"class", "value":"MISSENSE"}; }
-	/ "NONSENSE"i { return {"type":"class", "value":"NONSENSE"}; }
-	/ "NONSTART"i { return {"type":"class", "value":"NONSTART"}; }
-	/ "NONSTOP"i { return {"type":"class", "value":"NONSTOP"}; }
-	/ "FRAMESHIFT"i { return {"type":"class", "value":"FRAMESHIFT"}; }
-	/ "INFRAME"i { return {"type":"class", "value":"INFRAME"}; }
-	/ "SPLICE"i { return {"type":"class", "value":"SPLICE"}; }
-	/ "TRUNC"i { return {"type":"class", "value":"TRUNC"}; }
-        / "FUSION"i { return {"type":"class", "value":"FUSION"}; }
-        / letter:AminoAcid position:NaturalNumber string:String { return {"type":"name" , "value":(letter+position+string)};}
-        / letter:AminoAcid position:NaturalNumber { return {"type":"position", "value":parseInt(position)}; }
-	/ mutation_name:String { return {"type":"name", "value":mutation_name}; }
+	= "MISSENSE"i { return {"type":"class", "value":"MISSENSE", "info":{}}; }
+	/ "NONSENSE"i { return {"type":"class", "value":"NONSENSE", "info":{}}; }
+	/ "NONSTART"i { return {"type":"class", "value":"NONSTART", "info":{}}; }
+	/ "NONSTOP"i { return {"type":"class", "value":"NONSTOP", "info":{}}; }
+	/ "FRAMESHIFT"i { return {"type":"class", "value":"FRAMESHIFT", "info":{}}; }
+	/ "INFRAME"i { return {"type":"class", "value":"INFRAME", "info":{}}; }
+	/ "SPLICE"i { return {"type":"class", "value":"SPLICE", "info":{}}; }
+	/ "TRUNC"i { return {"type":"class", "value":"TRUNC", "info":{}}; }
+        / "FUSION"i { return {"type":"class", "value":"FUSION", "info":{}}; }
+        / letter:AminoAcid position:NaturalNumber string:String { return {"type":"name" , "value":(letter+position+string), "info":{}};}
+        / letter:AminoAcid position:NaturalNumber { return {"type":"position", "value":parseInt(position), "info":{"amino_acid":letter.toUpperCase()}}; }
```

Comment:
why is there this difference between this line and the line above? I.e. in the previous line the `value` is set as `(letter+position....` while here the letter suddenly needs to be in `info`.